include/crocoddyl/multibody/wrench-cone.hxx

```diff
@@ -142,9 +142,12 @@ void WrenchConeTpl<Scalar>::update() {
   Scalar mu = mu_;
   Scalar theta = Scalar(2) * M_PI / static_cast<Scalar>(nf_);
   if (inner_appr_) {
-    mu *= cos(theta / Scalar(2.));
+    mu *= cos(theta * Scalar(0.5));
```

Comment:
There's a thing about multiplication operations being more efficient than division operations depending on the processor. The other is never the case.